cmd/wallet-web/src/pages/Vaults.vue

```diff
@@ -9,22 +9,29 @@
     <div class="mb-8 w-full">
       <h3 class="text-neutrals-dark">{{ $t('Vaults.heading') }}</h3>
     </div>
-    <div class="px-6 w-64 h-auto bg-neutrals-white rounded-xl">
-      <div class="pt-5">
-        <div class="flex justify-center items-center w-12 h-12 bg-gradient-pink rounded-full">
-          <img class="w-6 h-5" src="@/assets/img/vaults.svg" alt="Vault Icon" />
-        </div>
-      </div>
-      <div class="pb-4 space-y-1">
-        <span class="block pt-4 text-lg font-bold"> {{ $t('Vaults.allVaults') }}</span>
-        <span class="block text-sm text-neutrals-medium">
-          {{
-            $tc('Vaults.foundCredentials', credentialsFound, {
-              credentialLength: credentialsFound,
-            })
-          }}
-        </span>
-      </div>
+    <div class="grid grid-cols-1 lg:grid-cols-3 gap-4 xl:gap-8">
+      <vault-card
+        color="pink"
+        type="regular"
+        :credential-found="
+          $tc('Vaults.foundCredentials', credentialsFound, {
+            credentialLength: credentialsFound,
+          })
+        "
+        :text="$t('Vaults.allVaults')"
+      />
+      <!--Issue-1198 Add flyout menu to default and otther vaults -->
+      <vault-card
+        color="purple"
+        type="regular"
+        :credential-found="
+          $tc('Vaults.foundCredentials', credentialsFound, {
+            credentialLength: credentialsFound,
+          })
+        "
+        :text="$t('Vaults.defaultVault')"
+      />
+      <vault-card color="purple" type="addVault" :text="$t('Vaults.addVault')" />
```

Comment:
Are we just adding these as hard-coded values for demonstration purposes for now? Eventually, they should be dynamically added based on the user's data, right?

cc @rolsonquadras

Reply:
All  vault, and default vault will always be added as default vault page as confirmed with mike, Add Vault will be dynamic which will be added after this pr